Components/Core/Inc/Timer.hpp

```diff
@@ -27,8 +34,10 @@ class Timer
 public:
 	Timer();
 
-	bool ChangePeriod(const uint32_t period);
-
+	bool ChangePeriod(const uint32_t period_ms);
+	bool StartTimer();
+	bool StopTimer();
+	TimerState GetState();
 	// WORK-IN-PROGRESS
```

Comment:
Add reset function, and a GetPeriod function, these are all useful utilities that could be useful.

Reply:
Also make a GetRemainingTime() function

Reply:
And SetAutoReload and GetAutoReload